mkt/developers/templates/developers/terms_standalone.html

```diff
@@ -0,0 +1,29 @@
+<html>
+  <head>
+    {{ css('mkt/devreg') }}
```

Comment:
It's a pretty simple way to add all the fonts and resets. I was shooting for simplicity over optimization, given the very specific nature of it.